roles/database/files/sql/creation/fworch-create-constraints.sql

```diff
@@ -1,13 +1,18 @@
-
-/* Create Alternate Keys */
 Alter Table "changelog_object" add Constraint "alt_key_changelog_object" UNIQUE ("abs_change_id");
 Alter Table "changelog_rule" add Constraint "alt_key_changelog_rule" UNIQUE ("abs_change_id");
 Alter Table "changelog_service" add Constraint "alt_key_changelog_service" UNIQUE ("abs_change_id");
 Alter Table "changelog_user" add Constraint "alt_key_changelog_user" UNIQUE ("abs_change_id");
+ALTER TABLE gw_interface DROP CONSTRAINT IF EXISTS gw_interface_routing_device_foreign_key;

```

Comment:
what is the purpose of drop statements in this file? As I understood, it is only used for clean install? So I would expect them in the upgrade file

Reply:
fixed